services/src/test/java/io/scalecube/services/ServiceTest.java

```diff
@@ -1,791 +1,791 @@
-        .port(port.incrementAndGet())
-        .seeds(gateway.cluster().address())
-        .services().service(new GreetingServiceImplB()).tag("Weight", "0.7").add()
-        .build()
-        .build();
-
-    CanaryService service = gateway.proxy()
-        .router(CanaryTestingRouter.class)
-        .api(CanaryService.class).create();
-
-    AtomicInteger count = new AtomicInteger(0);
-    AtomicInteger responses = new AtomicInteger(0);
-    CountDownLatch timeLatch = new CountDownLatch(1);
-    for (int i = 0; i < 100; i++) {
-      service.greeting("joe").whenComplete((success, error) -> {
-        responses.incrementAndGet();
-        if (success.startsWith("B")) {
-          count.incrementAndGet();
-          if ((responses.get() == 100) && (60 < count.get() && count.get() < 80)) {
-            timeLatch.countDown();
-          }
-        }
-      });
-    }
-
-    await(timeLatch, 1, TimeUnit.SECONDS);
-    assertTrue((responses.get() == 100) && (60 < count.get() && count.get() < 80));
-    
-    gateway.cluster().shutdown();
-    services1.cluster().shutdown();
-    services2.cluster().shutdown();
-  }
-
-  private GreetingService createProxy(Microservices gateway) {
-    return gateway.proxy()
-        .api(GreetingService.class) // create proxy for GreetingService API
-        .create();
-  }
-
-  private GreetingService createProxy(Microservices micro, Duration duration) {
-    return micro.proxy()
-        .api(GreetingService.class) // create proxy for GreetingService API
-        .timeout(duration)
-        .create();
-  }
-
-  private Microservices createProvider(Microservices gateway) {
-    return Microservices.builder()
-        .seeds(gateway.cluster().address())
-        .port(port.incrementAndGet())
-        .build();
-  }
-
-  private Microservices createSeed() {
-    return Microservices.builder()
-        .port(port.incrementAndGet())
-        .build();
-  }
-
-  private void await(CountDownLatch timeLatch, long timeout, TimeUnit timeUnit) {
-    try {
-      timeLatch.await(timeout, timeUnit);
-    } catch (InterruptedException e) {
-      throw new AssertionError();
-    }
-  }
-}
+package io.scalecube.services;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import io.scalecube.services.a.b.testing.CanaryService;
+import io.scalecube.services.a.b.testing.CanaryTestingRouter;
+import io.scalecube.services.a.b.testing.GreetingServiceImplA;
+import io.scalecube.services.a.b.testing.GreetingServiceImplB;
+import io.scalecube.testlib.BaseTest;
+import io.scalecube.transport.Message;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.time.Duration;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class ServiceTest extends BaseTest {

```

Comment:
Why does this class is shown as fully changed even so it doesn't contain any differences?